### PR TITLE
bsp: Use LVGL port for all BSPs with display.

### PR DESCRIPTION
**Pull request by espzav**
Used LVGL port component in all BSPs with display (except RGB display).

### CODE REVIEW

File: esp-box/esp-box.c [hunk 1]
```diff
@@ -18,47 +18,14 @@
 #include "bsp/esp-box.h"
 #include "esp_lcd_touch_tt21100.h"
 #include "esp_lvgl_port.h"
+#include "bsp_err_check.h"
```
Comment (tore-espressif) [inline esp-box/esp-box.c:22]: Could we also increase the default draw buffer size? base on our performance measuremens?

**Review by tore-espressif — APPROVED** [review]
LGTM!

Only Azure Kit and ESP-BOX don't have updated  versions

File: esp-box/esp-box.c (hunk 1 above)
Comment (espzav) [reply esp-box/esp-box.c:22]: I think, that we want to do it in some review before, but here it is: https://github.com/espressif/esp-bsp/pull/101#pullrequestreview-1238794515

Comment (tore-espressif) [reply esp-box/esp-box.c:22]: Ah, OK

**Comment by espzav** [comment]
@tore-espressif Thank you  for review!

**Pull request merged**